cliquet_docs/api/resource.rst

```diff
@@ -876,7 +876,7 @@ In a response, ``permissions`` contains the current permissions of the record
 
 .. note::
 
-    When a record is created or modified, the current :term`user id`
+    When a record is created or modified, the current :term:`user id`
     **is always added** among the ``write`` principals.
 
-`Read more about leveraging resource permissions <resource-permissions>`.
+:ref:`Read more about leveraging resource permissions <permissions>`.
```

Comment:
Just a warning that ``cliquet_docs`` is used in projects that uses it such as [Kinto](https://github.com/Kinto/kinto/blob/master/docs/conf.py#L31-L49) 

We need to make sure that if we change ``resource-permissions`` into ``permissions`` here, it doesn't breaks the references in the kinto docs as well.

I checked and it doesn't seems to be the case.